.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
Should we use version 14, or the latest version (e.g. `lts/*`)? I don't think it will matter too much, but I wondered if there was a reason we shouldn't use the latest version here.

Reply:
I keep the NodeJS version same as the Gutenberg repo.

Reply:
Ah OK, that sounds fine to me 👍